guides/source/getting_started.md

```diff
@@ -92,6 +92,11 @@ The version shown should be Rails 8.1.0 or higher.
 
 ### Creating Your First Rails App
 
+Note: In this guide, while specifying commands we use `rails` or `bin/rails`. 
```

Comment:
Instead of the note, it would be good to just fix all wrong occurrences if you see any.